config/initializers/menus.rb

```diff
@@ -76,7 +76,7 @@
             if: Proc.new { User.current.logged? }
   menu.push :administration,
             { controller: '/admin', action: 'index' },
-            if: Proc.new { User.current.admin? }
+            if: Proc.new { User.current.admin? || User.current.allowed_to_globally?(:add_user) }
```

Comment:
That `allowed_to_globally?` check will return true if the user is admin. Because of that, the first condition can be removed.

Reply:
I agree, but I found it confusing to the reader if the only check for an administration menu is some seemingly unrelated global permission. I'd be fine to delete, but there's also not much harm in leaving it explicitly in there is it?